R/io.R

```diff
@@ -260,7 +260,7 @@ createResource <- function(x, dir = FALSE, content = "") {
 
 uriExists <- function(x) {
   uri <- .parseURI(x)
-  if (uriIsLocal(x)) {
+  if (uriIsLocal(uri)) {
```

Comment:
Are you sure this doesn't break anything?

Reply:
I think yes, because of these reasons:
- `R CMD check rtracklayer` didn't report any error for any example or any unit test.
- `uriIsLocal()` is expecting an object of type `URI`.
- `uriExists()` is only [used once in the codebase](https://github.com/lawremi/rtracklayer/search?q=uriExists&unscoped_q=uriExists)  and that is working fine.

Reply:
Thanks for your diligence.